src/OpenLoco/src/Objects/Object.h

```diff
@@ -48,6 +48,14 @@ namespace OpenLoco
     constexpr size_t kMaxObjectTypes = 34;
     constexpr uint8_t kCargoTypeNull = 0xFF;
 
+    enum class SourceGame : uint8_t
+    {
+        custom,
+        vanilla, // Note: Most custom objects set this unfortunately so can't be trusted
```

Comment:
Vanilla is `2`, not `1`. I've also temporary disabled the `1` value since I don't think it exists in Loco/isn't valid (or if it is, I don't know what it's used for). This is what I've got in the editor:

![image](https://github.com/user-attachments/assets/ebc83c6d-913e-4027-b98e-c668c653e599)

Reply:
doh